src/routes/billingController.js

```
@@ -0,0 +1,690 @@
+
+    try {
+      // Extract common data from webhook
+      let jobData = {};
+
+      switch (jobType) {
+        case 'payment_failed':
+          const { data: failedUserSub } = await this.supabaseClient
+            .from('user_subscriptions')
+            .select('user_id')
+            .eq('stripe_customer_id', webhookData.customer)
+            .single();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)